embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java

```diff
@@ -67,6 +67,11 @@
         @ConfigDefault("null")
         public Optional<String> getAccessKeyId();
 
+        @Config("http_proxy")
+        @ConfigDefault("null")
+        public Optional<HttpProxy> getHttpProxy();
+        public void setHttpProxy(Optional<HttpProxy> httpProxy);
```

Comment:
The PR doesn't support it. If they want, they need to unset env `HTTP_PROXY` for Embulk and embulk-input-s3 for now. 

I think that we can take one of the following approaches. To be honest, 1st one was OK for me. 2nd one is reasonable for me now. 
1. Not support. Users need to unset env `HTTP_PROXY`
2. Provide an option to create `HttpProxy` from env variables specified by users
3. Stop creating `HttpProxy` from env variables.